### DISCUSSION TITLE
A page to see all possible Achevements

### DISCUSSION THREAD

**Discussion by RobertWHurst** [community/community#51611] [Other Feature Feedback, Questions, & Ideas]
Hi team GitHub!

Hope you are all doing well and having fun shipping it :shipit: 
GitHub is looking pretty snazzy these days.

### Question and Perhaps a Suggestion

Is there a page where all profile achievements can be seen? It would be great to have a page to see them all, and how rare they are. I've seen this done in a lot of games, and it helps motivate people to strive for those achievements, as well as let's them know you all have implemented more than just the 0-6 they see on their profile.

This all achievements view should probably be linked from or near the same UI component that lists the achievements on the profile.

Best wishes and thanks for all of the hard work!

**Comment by Neodevils** [comment]
Hello @RobertWHurst!

As far as I know, it's not existing right now but there's resources about it. There's even repository about it! 👀

**Comment by radar-blip** [comment]
List of all [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements). This is someones repo, not built into GitHub.
I think the hardest to get is/was the Mars 2020 Contributor Badge, though its not available anymore.
